core/services/chainlink/testdata/config-full.toml

```diff
@@ -0,0 +1,286 @@
+ContractSubscribeInterval = '1m0s'
+ContractTransmitterTransmitTimeout = '1m0s'
+DatabaseTimeout = '8s'
+KeyBundleID = '7a5f66bbe6594259325bf2b4f5b1a9c900000000000000000000000000000000'
+MonitoringEndpoint = 'test-mon-end'
+
+[OCR]
+ObservationTimeout = '11s'
+BlockchainTimeout = '3s'
+ContractPollInterval = '1h0m0s'
+ContractSubscribeInterval = '1m0s'
+DefaultTransactionQueueDepth = 12
+KeyBundleID = 'acdd42797a8b921b2910497badc5000600000000000000000000000000000000'
+MonitoringEndpoint = 'test-monitor'
+SimulateTransactions = true
+TransmitterAddress = '0xa0788FC17B1dEe36f057c42B6F373A34B014687e'
+
+[P2P]
+IncomingMessageBufferSize = 13
+OutgoingMessageBufferSize = 17
+
+[P2P.V1]
+AnnounceIP = '1.2.3.4'
+AnnouncePort = 1234
+BootstrapCheckInterval = '1m0s'
+BootstrapPeers = ['foo', 'bar', 'should', 'these', 'be', 'typed']
+DHTAnnouncementCounterUserPrefix = 4321
+DHTLookupInterval = 9
+ListenIP = '4.3.2.1'
+ListenPort = 9
+NewStreamTimeout = '1s'
+PeerID = '12D3KooWMoejJznyDuEk5aX6GvbjaG12UzeornPCBNzMRqdwrFJw'
+PeerstoreWriteInterval = '1m0s'
+
+[P2P.V2]
+AnnounceAddresses = ['a', 'b', 'c']
+Bootstrappers = ['1', '2', '3']
+DeltaDial = '1m0s'
+DeltaReconcile = '1s'
+ListenAddresses = ['foo', 'bar']
+
+[Keeper]
+CheckUpkeepGasPriceFeatureEnabled = true
+DefaultTransactionQueueDepth = 17
+GasPriceBufferPercent = 12
+GasTipCapBufferPercent = 43
+BaseFeeBufferPercent = 89
+MaximumGracePeriod = 31
+RegistryCheckGasOverhead = '90'
+RegistryPerformGasOverhead = '18446744073709551615'
+RegistrySyncInterval = '1h0m0s'
+RegistrySyncUpkeepQueueSize = 31
+TurnLookBack = 91
+TurnFlagEnabled = true
+
+[AutoPprof]
+Enabled = true
+ProfileRoot = 'prof/root'
+PollInterval = '1m0s'
+GatherDuration = '12s'
+GatherTraceDuration = '13s'
+MaxProfileSize = '1.00gb'
+CPUProfileRate = 7
+MemProfileRate = 9
+BlockProfileRate = 5
+MutexProfileFraction = 2
+MemThreshold = '1.00gb'
+GoroutineThreshold = 999
+
+[[EVM]]
+ChainID = '1'
+BalanceMonitorEnabled = true
+BlockBackfillDepth = 100
+BlockBackfillSkip = true
+BlockEmissionIdleWarningThreshold = '1h0m0s'
+ChainType = 'Optimism'
+EIP1559DynamicFees = true
+FinalityDepth = 42
+FlagsContractAddress = '0xae4E781a6218A8031764928E88d457937A954fC3'
+GasBumpPercent = 10
+GasBumpTxDepth = 6
+GasBumpWei = '100'
+GasEstimatorMode = 'L2Suggested'
+GasFeeCapDefault = '9223372036854775807'
+GasLimitDefault = '12'
+GasLimitMultiplier = '1.234'
+GasPriceDefault = '9223372036854775807'
+GasTipCapDefault = '2'
+GasTipCapMinimum = '1'
+HeadTrackerHistoryDepth = 15
+HeadTrackerMaxBufferSize = 17
+HeadTrackerSamplingInterval = '1h0m0s'
+LinkContractAddress = '0x538aAaB4ea120b2bC2fe5D296852D948F07D849e'
+LogBackfillBatchSize = 17
+LogPollInterval = '1m0s'
+MaxGasPriceWei = '281474976710655'
+MaxInFlightTransactions = 19
+MaxQueuedTransactions = 99
+MinGasPriceWei = '13'
+MinIncomingConfirmations = 13
+MinimumContractPayment = '9223372036854775807'
+NodeNoNewHeadsThreshold = '1m0s'
+NodePollFailureThreshold = 5
+NodePollInterval = '1m0s'
```

Comment:
Having just a `Node` prefix seems clear since it matches with `Nodes` :shrug: 
What about a `[Node]` table?
```suggestion
[Node]
NoNewHeadsThreshold = '1m0s'
PollFailureThreshold = 5
PollInterval = '1m0s'
```
Could refine the field names more too

Reply:
can also be written:
```suggestion
Node.NoNewHeadsThreshold = '1m0s'
Node.PollFailureThreshold = 5
Node.PollInterval = '1m0s'
```

Reply:
`NodePool`? :shrug: 
```suggestion
[NodePool]
NoNewHeadsThreshold = '1m0s'
PollFailureThreshold = 5
PollInterval = '1m0s'
```

Reply:
Yes this all looks like an improvement